cli/azd/pkg/auth/manager.go

```diff
@@ -821,6 +855,10 @@ func (m *Manager) saveSecret(tenantId, clientId string, ps *persistedSecret) err
 }
 
 type CredentialForCurrentUserOptions struct {
+	// Debug controls whether to enable OneAuth logging, including PII.
```

Comment:
Not in love with the `Debug` parameter here - but I understand the problem it's trying to solve.  Part of me feels like maybe to start instead we should control this via some set of env-vars like `AZD_ONEAUTH_DEBUG_LOG_LEVEL` and `AZD_ONEAUTH_DEBUG_INCLUDE_PII`.  In that world maybe we could log at info level without PII.

In the past we had sort of always been dumping log messages to `log` and then having `--debug` control on if they were visible or not.

Reply:
Yeah, this is perhaps a clumsy way to plumb `--debug` to the `oneauth` package. It should be possible to write a OneAuth log callback that calls a Go function with a string, which would be more complex but would obviate this field 🤔

Reply:
I wired up a callback OneAuth can use to send messages back to Go, enabling the `oneauth` package to follow the "just `log` everything all the time" convention. Also, I disabled PII logging so there's no need to design an opt-in gesture for that at this time.